back/engines/commercial/content_builder/spec/acceptance/layouts_spec.rb

```diff
@@ -0,0 +1,406 @@
+                    id: layout.id,
+                    type: 'content_builder_layout',
+                    attributes: {
+                      craftjs_jsonmultiloc: {
+                        'da-DK': {
+                          ROOT: {
+                            type: {
+                              resolvedName: 'Container'
+                            }
+                          }
+                        },
+                        'kl-GL': {
+                          ROOT: {}
+                        }
+                      },
+                      enabled: true,
+                      code: code,
+                      created_at: match(time_regex),
+                      updated_at: match(time_regex)
+                    }
+                  }
+                }
+              )
+              created_at = json_response.dig(:data, :attributes, :created_at)
+              updated_at = json_response.dig(:data, :attributes, :updated_at)
+              expect(updated_at).to be > created_at
+            end
+          end
+        end
+
+        describe 'updating multiple locales' do
+          let! :layout do
+            create(:layout)
+          end
+          let :craftjs_jsonmultiloc do
+            {
+              'nl-BE': {
+                ROOT: {
+                  type: {
+                    resolvedName: 'Container'
+                  }
+                }
+              },
+              'kl-GL': {
+                ROOT: {}
+              }
+            }
+          end
+
+          post 'web_api/v1/projects/:project_id/content_builder_layouts/:code/upsert' do
+            example_request 'Update multiple locales of a layout of a project' do
+              expect(status).to eq 200
+              json_response = json_parse(response_body)
+              expect(json_response).to include(
+                {
+                  data: {
+                    id: layout.id,
+                    type: 'content_builder_layout',
+                    attributes: {
+                      craftjs_jsonmultiloc: {
+                        'nl-BE': {
+                          ROOT: {
+                            type: {
+                              resolvedName: 'Container'
+                            }
+                          }
+                        },
+                        'kl-GL': {
+                          ROOT: {}
+                        }
+                      },
+                      enabled: true,
+                      code: code,
+                      created_at: match(time_regex),
+                      updated_at: match(time_regex)
+                    }
+                  }
+                }
+              )
+              created_at = json_response.dig(:data, :attributes, :created_at)
+              updated_at = json_response.dig(:data, :attributes, :updated_at)
+              expect(updated_at).to be > created_at
+            end
+          end
+        end
+
+        describe 'disabling a locale' do
```

Comment:
Regarding the presence of locales in the craftjs_jsonmultiloc, I don't see a difference between "disabling a locale" and "enabling a locale". Did you perhaps mean "disabling a layout"?

Reply:
Good catch! Of course it should be "enabling a layout" and "disabling a layout".
Fixed.